kernel/src/util/iovec.rs

```diff
@@ -225,6 +263,20 @@ impl MultiWrite for VmWriterArray<'_> {
     fn sum_lens(&self) -> usize {
         self.0.iter().map(|vm_writer| vm_writer.avail()).sum()
     }
+
+    fn skip(&mut self, mut nbytes: usize) {
+        assert!(nbytes <= self.sum_lens());
```

Comment:
Remove the assertion for the writer as well since the assertion for the reader has been dropped.